### PR TITLE
Fixes maximum call stack issue by directly referencing attributes.

### PR DESCRIPTION
**Pull request by brianpmarks**
Attempt to fix trinketapp/trinket#1772.

### CODE REVIEW

File: __init__.js [hunk 1]
```diff
@@ -169,12 +171,13 @@ var $builtinmodule = function(name) {
...
-        var options = Sk.ffi.remapToJs(self.$d);
-        for(var key in options) {
-          if (options.hasOwnProperty(key)) {
-            self.instance._options[key] = options[key];
-          }
+        var i, key;
+
+        for (i = 0; i < KWARGS.length; i++) {
+          key = KWARGS[i];
+          self.instance._options[key] = Sk.ffi.remapToJs(self.tp$getattr(key));
```
Comment (albertjan) [inline __init__.js:178]: does this work when the key is a js string instead of a skulpt string?

Comment (brianpmarks) [reply __init__.js:178]: It does. Would it be better if key were made a `Sk.builtin.str`?

Comment (albertjan) [reply __init__.js:178]: No if it works it should stay this way 😄

**Comment by eah13** [comment]
Not sure about the big picture but this fixes the bug in question so 🚢

**Pull request merged**